apps/server/src/modules/authentication/controllers/dto/migration.response.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class MigrationResponse {
+	@ApiProperty()
+	sourceSystem: string;
+
+	@ApiProperty()
+	targetSystem: string;
+
+	@ApiProperty()
+	mandatory: boolean;
```

Comment:
Shouldn't we name boolean properties with a prefix like `is` or `has`, e.g. `isMandatory`?